core/common/models/Settings.php

```diff
@@ -0,0 +1,303 @@
+{
+
+    /**
+     *
+     * @var integer
+     */
+    protected $id;
+
+    /**
+     *
+     * @var string
+     */
+    protected $name;
+
+    /**
+     *
+     * @var string
+     */
+    protected $value;
+
+    /**
+     *
+     * @var string
+     */
+    protected $note;
+
+    public function initialize()
+    {
+        \Phalcon\Mvc\Model::setup([
+           'notNullValidations' => false
+        ]);
+    }
+
+    /**
+     * Method to set the value of field id
+     *
+     * @param  integer $id
+     * @return $this
+     */
+    public function setId($id)
+    {
+        $this->id = $id;
+
+        return $this;
+    }
+
+    /**
+     * Method to set the value of field name
+     *
+     * @param  string $name
+     * @return $this
+     */
+    public function setName($name)
+    {
+        $this->name = $name;
+
+        return $this;
+    }
+
+    /**
+     * Method to set the value of field value
+     *
+     * @param  string $value
+     * @return $this
+     */
+    public function setValue($value)
+    {
+        $this->value = $value;
+
+        return $this;
+    }
+
+    /**
+     * Method to set the value of field note
+     *
+     * @param  string $note
+     * @return $this
+     */
+    public function setNote($note)
+    {
+        $this->note = $note;
+
+        return $this;
+    }
+
+    /**
+     * Returns the value of field id
+     *
+     * @return integer
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Returns the value of field name
+     *
+     * @return string
+     */
+    public function getName()
+    {
+        return $this->name;
+    }
+
+    /**
+     * Returns the value of field value
+     *
+     * @return string
+     */
+    public function getValue()
+    {
+        return $this->value;
+    }
+
+    /**
+     * Returns the value of field note
+     *
+     * @return string
+     */
+    public function getNote()
+    {
+        return $this->note;
+    }
+
+    /**
+     * Returns table name mapped in the model.
+     *
+     * @return string
+     */
+    public function getSource()
+    {
+        return 'settings';
+    }
+
+    /**
+     * Allows to query a set of records that match the specified conditions
+     *
+     * @param  mixed $parameters
+     * @return Settings[]
+     */
+    public static function find($parameters = null)
+    {
+        return parent::find($parameters);
+    }
+
+    /**
+     * Allows to query the first record that match the specified conditions
+     *
+     * @param  mixed $parameters
+     * @return Settings
+     */
+    public static function findFirst($parameters = null)
+    {
+        return parent::findFirst($parameters);
+    }
+
+    /**
+     * Independent Column Mapping.
+     * Keys are the real names in the table and the values their names in the application
+     *
+     * @return array
+     */
+    public function columnMap()
+    {
+        return array(
```

Comment:
It created default by phalcon tool, you can fix this in phalcon tool to use late